### PR TITLE
[4.0][webservice][com_content] add more query param to GET

### PR DESCRIPTION
**Pull request by alikon**
### Summary of Changes
add more query parameters

### Testing Instructions

 `GET http://localhost/<yoursite>/api/v1/content/article?condition=0&language=*&author=1`

### Expected result

you can get result filtered for condition,author and language

### Actual result
you can get all content or 1 content

### CODE REVIEW

**Comment by SharkyKZ** [comment]
Is it necessary to have a method for each input variable?

**Comment by alikon** [comment]
No, i think more readable this way

**Comment by brianteeman** [comment]
Please add the documentation required flag

File: api/components/com_content/Controller/ArticlesController.php [hunk 1]
```diff
@@ -65,4 +65,77 @@ protected function save($recordKey = null)
...
+	 * Get author id from input
+	 *
+	 * @return string
+	 *
+	 * @since 4.0
+	 */
+	private function getAuthorIdFromInput()
+	{
+		return $this->input->exists('author') ?
+			$this->input->get('author') : $this->input->post->get('author');
```
Comment (SharkyKZ) [inline api/components/com_content/Controller/ArticlesController.php]: Can be simplified:

return $this->input->get('author', $this->input->post->get('author'));

**Comment by SharkyKZ** [comment]
@alikon That's totally unnecessary code repetition, IMO.

File: api/components/com_content/Controller/ArticlesController.php (hunk 1 above)
Comment (SharkyKZ) [inline api/components/com_content/Controller/ArticlesController.php]: Can be simplified:

```
return $this->input->get('author', $this->input->post->get('author'));
```

Comment (alikon) [reply api/components/com_content/Controller/ArticlesController.php]: ok, sounds good
done

**Pull request closed without merging**